site/assets/scss/_content.scss

```diff
@@ -32,8 +32,8 @@
   // Override Bootstrap defaults
   > .table,
   > .table-responsive .table {
-    // Boosted mod: remove unwanted rules
-    // Prevent breaking of code
```

Comment:
This comment comes from Bs

Reply:
Fixed via https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1757/commits/0e7754df063a594f72f75455352b4472ea9976ee